docs/cheatsheet/types.rst

```diff
@@ -0,0 +1,107 @@
+.. _ref_cheatsheet_types:
+
+Types
+=====
+
+Define an abstract type:
+
+.. code-block:: sdl
+
+    abstract type HasImage {
```

Comment:
Need to add another cheat sheet page -- migrations, where you can explain what sdl is and how to use it. Then link that page from this page. For a new user it's not obvious how to use SDL.

Reply:
Added.